setup.py

```diff
@@ -1,24 +1,29 @@
+import os
 from codecs import open  # To use a consistent encoding
-from os import path
 
 from setuptools import setup
 
-here = path.abspath(path.dirname(__file__))
+here = os.path.abspath(os.path.dirname(__file__))
```

Comment:
just a small question, why not `os.getcwd()`?

Reply:
Users may be in a different directory from this script.

For example, we could execute build command from the parent directory:

```bash
python aftership-sdk-python/setup.py ...
```

In this scenario, use `os.getpwd` would result in the wrong directory.

Usually, we location script location usually use `os.path.dir(__file__)` instead of `os.get_pwd`

Reply:
ah, ok